account/manager.go

```diff
@@ -1200,20 +1251,17 @@ func (m *Manager) spendAccount(ctx context.Context, account *Account,
 		spendPkg *spendPackage
 		err      error
 	)
-	switch witnessType {
-	case expiryWitness:
-		// TODO(wilmer): Support modifications through the expiry path.
-		// This will require a new account expiration.
-		if !isClose {
-			return nil, nil, errors.New("modifications for expired " +
-				"accounts are not currently supported")
-		}
-
+	switch {
+	// Only force the expiry path once the account has expired and the
+	// account is being closed.
+	case witnessType == expiryWitness && isClose:
```

Comment:
The account is already expired at this point. This is to ensure any account modifications after the account's been expired always go through the auctioneer so that it can learn about the changes being made.

Reply:
shouldn't `expiryWitness && !isClose`  be an invalid combination?

Reply:
Not necessarily, the multi-sig spend path can still be used after the expiration as long as the auctioneer allows it, which it will only do so in the event of a renewal.

Reply:
Yeah, but if you use `expiryWitness` with `isClose == false` it won't go through the auctioneer, and the client is trying to not close the account? I don't understand